### DISCUSSION TITLE
Welcome to CommandSender Discussions!

### DISCUSSION THREAD

**Discussion by Zayik** [Zayik/CommandSender#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by I8NHJ** [comment]
Hi Zayik. Thanks for the plug-in and for opening this discussion space.

**Comment by Shounteal** [comment]
Thanks for setting this up!! I've been going nuts trying to figure out how to control my lights with the stream deck and this was really simple to set up!

**Comment by vandenkm** [comment]
Greetings Zayik!

I found your plug-in referenced by some HAM Radio software called N1MM Logger+  It has already been incredibly useful!  Thanks for putting it out there.

**Reply by Zayik** [reply]
Glad it's working great for y'all.

**Comment by AudioFlavor** [comment]
The description of how to send hexadecimal characters is inaccurate on the Elgato page. The 'slash' preceding the 'x' needs to be a '\' (backslash) whereas the description today denotes a '/' (forward slash). Also, is there any guidance available on how the states function?
